CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 * [#71](https://github.com/alexa-js/alexa-app-server/pull/71), [#68](https://github.com/alexa-js/alexa-app-server/issues/68): Fixed log output containing multiple slashes - [@tejashah88](https://github.com/tejashah88).
 * [#72](https://github.com/alexa-js/alexa-app-server/pull/72): Use `path.join` for constructing relative paths - [@dblock](https://github.com/dblock).
 * [#74](https://github.com/alexa-js/alexa-app-server/pull/74): Added locale selector to test page - [@siedi](https://github.com/siedi).
+* Changed endpoint message to use app name to match route
```

Comment:
Would you be so kind to make this look similar to other lines? Everything is on the same line, there's a `-`, a period at the end, etc.